great_expectations/data_context/data_context_v3.py

```diff
@@ -38,21 +35,10 @@ def get_config(self, mode="typed"):
             return config.commented_map
 
         elif mode == "dict":
-            return dict(config.commented_map)
+            return config.to_dict()
 
         elif mode == "yaml":
-            commented_map = copy.deepcopy(config.commented_map)
-            commented_map.update(dataContextConfigSchema.dump(config))
-
-            stream = StringIO()
-            yaml.dump(commented_map, stream)
-            yaml_string = stream.getvalue()
-
-            # print(commented_map)
-            # print(commented_map.__dict__)
-            # print(str(commented_map))
-            return yaml_string
-            # config.commented_map.update(dataContextConfigSchema.dump(self))
+            return config.to_yaml_str()
```

Comment:
Can we just remove this entire method so we don't change the behavior of get_config?

Reply:
Or, should this become he base implementation?

Reply:
This was done (committed).